Refresh.GameServer/Database/GameDatabaseProvider.cs

```diff
@@ -33,45 +34,66 @@ protected GameDatabaseProvider(IDateTimeProvider time)
         this._time = time;
     }
 
-    protected override ulong SchemaVersion => 149;
+    protected override ulong SchemaVersion => 154;
 
     protected override string Filename => "refreshGameServer.realm";
     
-    protected override List<Type> SchemaTypes { get; } = new()
-    {
-        typeof(GameUser),
-        typeof(UserPins),
-        typeof(Token),
+    protected override List<Type> SchemaTypes { get; } =
+    [
+        typeof(RequestStatistics),
+        typeof(SequentialIdStorage),
+
+        // recent activity
+        typeof(Event),
+
+        // announcements
+        typeof(GameAnnouncement),
+
+        // contests
+        typeof(GameContest),
+
+        // photos
+        typeof(GamePhoto),
```

Comment:
These types should be left at the top. I don't think we need this degree of categorization to the point of organizing one-by-one. They can be left self-explanatory.